.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+# Checklist
+
+- Branch
```

Comment:
Ah, one thing to add: if serialization changes (eg there are diffs in the golden test files), we need to confirm any user-facing consequences (eg replay from genesis block) are intentional

Reply:
Added